cfpb/cfgov-refresh
# Rate checker: Clean up `event` variable

There was a mix of `event` and `evt` used for event names. Unfortunately, one spot used one but passed in the other, leading to this error:

<img width="350" alt="screen shot 2018-12-19 at 9 28 50 am" src="https://user-images.githubusercontent.com/704760/50226174-84c16c80-0370-11e9-8e60-3c7b368883cb.png">

## Changes

- Fixes misnamed `event` variable name.
- ~~Standardizes on `evt` name for event object within rate explorer script.~~
- Standardizes on `event` name for event object within rate explorer script.

## Testing

In Firefox 63 or earlier:

1. `gulp clean && gulp build`
2. Pull this branch and visit /owning-a-home/explore-rates/
3. Change the house price input field and see that there are no errors in dev console. Do the same in https://www.consumerfinance.gov/owning-a-home/explore-rates/ and see that there are.
